src/hardware/mod.rs

```diff
@@ -53,10 +57,32 @@ pub type NetworkManager = smoltcp_nal::shared::NetworkManager<
 pub type EthernetPhy = hal::ethernet::phy::LAN8742A<hal::ethernet::EthernetMAC>;
 
 /// System timer (RTIC Monotonic) tick frequency
-pub const MONOTONIC_FREQUENCY: u32 = 1_000;
+pub const MONOTONIC_FREQUENCY: u32 = 10_000;
```

Comment:
Why do you need 10 kHz here?
If you change this then we have to double check the network stack timing as that expects millisecond ticks IIRC.

Reply:
Ok. The problem is that this sets the highest possible Ltc2320 sample rate. Otherwise I'd have to eg. use a timer again instead of rtic scheduling.

Reply:
Ok. Put (a) a comment why, (b) a TODO to clear this up and leave it like this for now. Later this may then require a timer.